app/controllers/client_api/users/invitations_controller.rb

```diff
@@ -0,0 +1,34 @@
+module ClientApi
+  module Users
+    class InvitationsController < Devise::InvitationsController
+      before_action :check_invite_users_permission, only: :invite_users
+
+      def invite_users
+        invite_service =
+          ClientApi::InvitationsService.new(user: current_user,
+                                            team: @team,
+                                            role: params['user_role'],
+                                            emails: params[:emails])
+        invite_results = invite_service.invitation
+        success_response(invite_results)
+      end
+
+      def success_response(invite_results)
+        respond_to do |format|
+          format.json do
+            render template: '/client_api/users/invite_users',
+                   status: :ok,
+                   locals: { invite_results: invite_results, team: @team }
+          end
+        end
+      end
+
+      private
+
+      def check_invite_users_permission
+        @team = Team.find_by_id(params[:team_id])
+        render_403 if @team && !is_admin_of_team(@team)
```

Comment:
if we render_403 you will get a html response. Probably is better to handle this somehow different like send 422 and a message that the user doesn't have permissions

Reply:
:+1: